vector/src/main/java/im/vector/app/features/spaces/NewSpaceSummaryController.kt

```diff
@@ -50,32 +51,25 @@ class NewSpaceSummaryController @Inject constructor(
                 nonNullViewState.spaces,
                 nonNullViewState.selectedSpace,
                 nonNullViewState.rootSpacesOrdered,
-                nonNullViewState.homeAggregateCount
+                nonNullViewState.homeAggregateCount,
+                nonNullViewState.expandedStates,
         )
     }
 
     private fun buildGroupModels(
             spaceSummaries: List<RoomSummary>?,
             selectedSpace: RoomSummary?,
             rootSpaces: List<RoomSummary>?,
-            homeCount: RoomAggregateNotificationCount
+            homeCount: RoomAggregateNotificationCount,
+            expandedStates: Map<String, Boolean>,
     ) {
-        val host = this
         newSpaceListHeaderItem {
             id("space_list_header")
         }
```

Comment:
This is removed in another PR (https://github.com/vector-im/element-android/pull/6877) so I didn't bother abstracting this